bin/rad-diff

```diff
@@ -5,23 +5,19 @@
 (load! (find-module-file! "prelude/io-utils.rad"))
 (load! (find-module-file! "monadic/project.rad"))
 
-(import prelude/cmd-parsing :unqualified)
 (import prelude/validation :as 'validation)
 
 (def machine-name (fn [] (get-rsm-of-type! :rad-diff)))
 
-(def valid-list-options
-  ["--pending" "--retracted" "--accepted" "--rejected" "--fancy"])
-
-(def list-opts-help
-  (foldl (fn [a opt] (string-append a " [" opt "]")) "" valid-list-options))
+(def valid-states
+  ["pending" "retracted" "accepted" "rejected"])
 
 (def help
   (string-append
   "rad-diff - Radicle Diff CLI
 
    Usage:
-        rad-diff list " list-opts-help "
+        rad-diff list [--filter-by-state | --state | -s <state>] [--fancy]
```

Comment:
This is a bit confusing. It looks like `--state` and `--filter-by-state` do not take arguments. We should also document what the possible states are.

Reply:
Yes, I thought the same, but wasn't sure how to fit all this info into the section.
Would this work? `rad-diff list [(--filter-by-state | --state | -s) (proposed | retracted | accepted | rejected)]`

Reply:
Yes, that would work. Maybe @juliendonck wants to chime in.

Reply:
I talked with @juliendonck and we decided to not list all states in the help text for simplicity. Instead the error message for invalid state arguments will return a list of valid states.